<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SocketTCPServer</class>
 <widget class="QDialog" name="SocketTCPServer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>258</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SocketTCPServer</string>
  </property>
  <widget class="QLabel" name="m_portLabel">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>12</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>端口号：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_portLineEdit">
   <property name="geometry">
    <rect>
     <x>67</x>
     <y>13</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="m_initSocketBtn">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>初始化Socket</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="m_inputTextEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>211</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="m_inputLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>输入内容：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_sendData">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>50</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_recvdataLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>接收数据：</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="m_recvDataTextEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>211</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
